--- jamvm-1.3.3/configure.ac.orig	2005-09-13 01:41:06.000000000 +0200
+++ jamvm-1.3.3/configure.ac	2005-10-22 03:44:00.243502664 +0200
@@ -55,7 +55,7 @@
 AC_CANONICAL_HOST
 
 case "$host" in
-i[[456]]86-*-linux*) host_cpu=i386 host_os=linux ;;
+i[[4567]]86-*-linux*) host_cpu=i386 host_os=linux ;;
 i[[456]]86-*-openbsd*) host_cpu=i386 host_os=linux libdl_needed=no ;;
 x86_64-*-linux*) host_os=linux ;;
 arm*-*-linux*) host_cpu=arm host_os=linux ;;
