--- jamvm-1.4.0/configure.ac.orig	2005-11-20 08:18:33.000000000 +0100
+++ jamvm-1.4.0/configure.ac	2005-11-27 01:32:27.752386896 +0100
@@ -55,7 +55,7 @@
 AC_CANONICAL_HOST
 
 case "$host" in
-i[[456]]86-*-linux*) host_cpu=i386 host_os=linux ;;
+i[[4567]]86-*-linux*) host_cpu=i386 host_os=linux ;;
 i[[456]]86-*-kfreebsd*) host_cpu=i386 host_os=linux ;;
 i[[456]]86-*-openbsd*) host_cpu=i386 host_os=linux libdl_needed=no ;;
 x86_64-*-linux*) host_os=linux ;;
