--- jamvm-1.4.3/configure.ac.orig	2006-05-22 01:08:26.000000000 +0200
+++ jamvm-1.4.3/configure.ac	2006-05-27 00:57:30.424819250 +0200
@@ -56,7 +56,7 @@
 AC_CANONICAL_HOST
 
 case "$host" in
-i[[3456]]86-*-linux*) host_cpu=i386 host_os=linux ;;
+i[[34567]]86-*-linux*) host_cpu=i386 host_os=linux ;;
 i[[3456]]86-*-kfreebsd*) host_cpu=i386 host_os=linux ;;
 i386-*-openbsd*) host_os=bsd libdl_needed=no ;;
 i386-*-freebsd*) host_os=bsd libdl_needed=no ;;
