--- jamvm-1.2.0/src/dll.c.orig	2004-08-30 12:41:50.000000000 +0200
+++ jamvm-1.2.0/src/dll.c	2004-10-20 21:13:02.071328216 +0200
@@ -256,7 +256,11 @@
 
 char *getDllPath() {
     char *env = getenv("LD_LIBRARY_PATH");
-    char *cp_path = CLASSPATH_INSTALL_DIR"/lib/classpath";
+#if defined(__x86_64__) || defined(__arch64__)
+    char *cp_path = CLASSPATH_INSTALL_DIR"/lib64";
+#else
+    char *cp_path = CLASSPATH_INSTALL_DIR"/lib";
+#endif
 
     if(env) {
         char *buff = sysMalloc(strlen(env) + strlen(cp_path) + 2);
@@ -271,7 +275,7 @@
 char *getDllName(char *path, char *name) {
    char *buff = sysMalloc(strlen(path) + strlen(name) + 8);
 
-   sprintf(buff, "%s/lib%s.so", path, name);
+   sprintf(buff, "%s/lib%s.so.0", path, name);
    return buff;
 }
 
